frontend/apps/app/next.config.ts

```diff
@@ -96,7 +96,21 @@ export default withSentryConfig(nextConfig, {
   // Hides source maps from generated client bundles
   sourcemaps: {
     disable: false,
-    assets: '.next',
+    assets: [
+      '.next/**/*.js',
+      '.next/**/*.js.map',
+      '.next/static/chunks/**/*.js',
+      '.next/static/chunks/**/*.js.map',
+      '.next/static/runtime/*.js',
+      '.next/static/runtime/*.js.map',
+      '.next/server/pages/**/*.js',
+      '.next/server/pages/**/*.js.map',
+      '.next/server/chunks/*.js',
+      '.next/server/chunks/*.js.map',
+      '.next/app-build-manifest.json',
+      '.next/build-manifest.json',
+    ],
+    ignore: ['node_modules/**/*'],
```

Comment:
Why is @liam-hq/app the only assets clarification needed?

Reply:
liam-app had 100~177files uploaded

![ss 2970](https://github.com/user-attachments/assets/7bc97894-831c-4a5e-a01b-246e2df23500)

Reply:
@FunamaYukina 
I don't think it's a good idea to delete a file just because "it might be large". It's best to understand what's included in the source map and how it's used before you organize it.

Reply:
You are right, I will create a new issue as I have not been able to look into the docs ``sourcemap`` yet!🙏

Reply:
I have found that another way to handle the issue is better, so I will respond with the following pull request🙏
- https://github.com/liam-hq/liam/pull/977